modules/creation/tutorial.md

```diff
@@ -132,7 +132,7 @@ $this->bootstrap = true;
 
 This section handles the relationship with the module and its environment (namely, PrestaShop):
 
-- The `need_instance` attribute Indicates whether to load the module's class when displaying the "Modules" page in the back office. If set at 0, the module will not be loaded, and therefore will spend less resources to generate the "Modules" page. If your module needs to display a warning message in the "Modules" page, then you must set this attribute to `1`.
+- The `need_instance` attribute Indicates whether to load the module's class when displaying the "Modules" page in the back office. If set at 0, the module will not be loaded, and therefore will spend less resources to generate the "Modules" page. If your module needs to display a warning message in the "Modules" page, then you must set this attribute to `1`. This is also necessary for payment and delivery modules to display checkboxes on the payment method preferences page.
```

Comment:
I'm not sure if we should give informations about specific modules here 🤔 

Maybe we could add something here?  https://devdocs.prestashop.com/1.7/modules/payment/